documentation/src/getting_started.md

```diff
@@ -1,19 +1,100 @@
+  - Many mirrorless (non-DSLR) cameras may work fine anyway due to live view mode being activated automatically (e.g. Sony α6400).
+  - The camera handling code is written with future special cases for different brands and models in mind, however we do not have any devices available for development and testing currently.
+  - **Workaround**: Try the different options available in the settings screen. If these don't work, your help might needed to fix the issue if possible at all. Please let us know by [raising an issue on GitHub](https://github.com/h3x4d3c1m4l/momento-booth/issues/new). Be sure to first check whether your model is listed on the [gPhoto2 compatibility list](http://www.gphoto.org/proj/libgphoto2/support.php). We might be able to expand the compatibility of the application with some remote testing. No guarantees however.
+
+## Minimal hardware (mostly useful for testing/development)
+
+A very basic (but useful for experimenting and developing) setup could consist of:
+
+- A computer
+  - OS either:
+    - Windows 10/11
+      - Windows 7/8/8.1 might work, but untested
+    - Linux
+      - Ubuntu 20.04 or newer, other recent distros should also be fine
+    - macOS
+      - Only tested on macOS 13 (Ventura), older versions might work but untested
+  - CPU: Intel Core i5 (2nd gen) or equivalent
+    - Older might work, but untested
+  - GPU: Integrated or dedicated graphics by Intel/AMD/NVIDIA
+    - Anything > 2010 should work fine, a little older may also work
+    - Software rendering might work but untested
+  - Memory: 4 GB
+  - Disk: HDD with at least some space left for photo storage
+  - Monitor: Anything
+- Webcam
+
+```mermaid
+graph LR
+    C{Computer} -- "USB (or Integrated)" --> W(Webcam)
+```
+
+## Recommended hardware
+
+- A computer
+  - OS either:
+    - Windows 10/11
+    - Linux
+      - Ubuntu 20.04 or newer, other recent distros should also be fine
+    - macOS 13
+  - CPU: Intel Core i3 (6th gen) or equivalent
+  - GPU: Integrated or dedicated graphics by Intel/AMD/NVIDIA
+  - Memory: 8 GB
+  - Disk: SSD with enough space left for photo storage
+  - Monitor: At least 1920x1080 with touch
+- Digital camera
+  - Live view: Supported directly by gPhoto2, or using HDMI out and a USB HDMI capture device (which can act like a webcam)
+  - Capturing: Supported directly by gPhoto2
+- Optional: Photo printer
+
+```mermaid
+graph LR
+    C{Computer} -- USB --> A(Camera)
+    H["HDMI capture device (optional)"] -- USB --> C{Computer}
+    A -- HDMI --> H
+    M -- "USB (touchscreen)" --> C
+    C{Computer} -- VGA/DVI/HDMI/DP --> M(Monitor)
+    C{Computer} -- "USB/(W)LAN" --> P(Photo printer)
+
+```
+
+## Running the application
+
+```admonish warning
+Make sure to use good quality USB and HDMI cables and preferably connect any USB devices without extension cords. There have been reports of freezes in camera handling due to bad cables.
+```
```

Comment:
Not part of running the application.

Reply:
Moved that one to the new Hardware Compatibility page.